File: JS/JS  Fundamentals/04_Functions/lab/grades.js
 function solve(grade) {
    let description = '';
    if (grade < 3) {

        description = 'Fail';
    } else if (grade >= 3 && grade < 3.5) {
        description = 'Poor';

    } else if(grade >= 3.5 && grade < 4.5) {
        description = 'Good';

    } else if(grade >= 4.5 && grade < 5.5) {
        description = 'Very good';
    } else {
        description = 'Exellent';
    }
    
    console.log(`${description} (${grade.toFixed(2)})`)
 }

 solve(3.33)
 solve(4.50)
 solve(2.99)

File: Python/Fundamentals/04_Functions/Lab/grades.py
grade = float(input())


def grades(a) -> object:
    if 2 <= a <= 2.99:
        print('Fail')
    elif 3 <= a <= 3.49:
        print('Poor')
    elif 3.5 <= a <= 4.49:
        print('Good')
    elif 4.5 <= a <= 5.49:
        print('Very Good')
    elif 5.5 <= a <= 6:
        print('Excellent')


grades(grade)
